### ISSUE TITLE
Add possibility to register onchange event

### ISSUE THREAD

**Issue by honza-klu** [dmnfarrell/pandastable#119]
It would be nice be able to register callback that would be called after any change in data.  It could be called from tableChanged. Is it right solution? I can prepare patch. Is there any other way to get this functionality?

**Comment by dmnfarrell** [comment]
Yes that sounds ok. You could test it and make a pull request if its working ok.

**Comment by dmnfarrell** [comment]
Did you manage to implement this?

**Comment by honza-klu** [comment]
Not yet, but it is on my todo list for weekend ...

út 19. 3. 2019 v 13:12 odesílatel Damien Farrell <notifications@github.com>
napsal:
